### PR TITLE
Add SPV_EXT_replicated_composites

### PR DESCRIPTION
**Pull request by kpet**
Headers: https://github.com/KhronosGroup/SPIRV-Headers/pull/431

### CODE REVIEW

**Review by Naghasan — APPROVED** [review]

File: extensions/EXT/SPV_EXT_replicated_composites.asciidoc [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+2+^| Capability ^| Depends On
+| 6024 | *ReplicatedCompositesEXT* +
+Uses **OpConstantCompositeReplicateEXT**, **OpSpecConstantCompositeReplicateEXT**, or **OpCompositeConstructReplicateEXT** |
+|====
+--
+
+Instructions
+~~~~~~~~~~~~
+
+Add the following new instructions:
```
Comment (jeffbolznv) [inline extensions/EXT/SPV_EXT_replicated_composites.asciidoc:90]: I think we usually indicate which section each new instruction should go it.

Comment (kpet) [reply extensions/EXT/SPV_EXT_replicated_composites.asciidoc]: Yup, now done.

File: extensions/EXT/SPV_EXT_replicated_composites.asciidoc [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+*ReplicatedCompositesEXT*
+| 4 | 4462
+| '<id> Result Type'
+| 'Result <id>'
+| '<id> Value'
+|======
+
+[cols="5*1"]
+|======
+4+|[[OpCompositeConstructReplicateEXT]]*OpCompositeConstructReplicateEXT* +
```
Comment (alan-baker) [inline extensions/EXT/SPV_EXT_replicated_composites.asciidoc:139]: This should be in Composite Instructions

**Review by alan-baker — APPROVED** [review]

**Review by jeffbolznv — APPROVED** [review]

**Comment by bashbaug** [comment]
Merging as discussed in the May 29th teleconference.

**Pull request merged**